Add to F-Droid

...as I don't see any closed source deps this should be fine, right?

Some thoughts
* start [Tagging your releases](https://github.com/urbandroid-team/dontkillmy-app/releases)
* add the app metadata _(summary/description/images/changelog/etc)_ in a [Fastlane](https://gitlab.com/snippets/1895688) or a [Triple-T](https://gitlab.com/snippets/1901490) folder structure


With this tentative metadata:
```
Categories:
  - System
License: Apache-2.0
SourceCode: https://github.com/urbandroid-team/dontkillmy-app
IssueTracker: https://github.com/urbandroid-team/dontkillmy-app/issues

RepoType: git
Repo: https://github.com/urbandroid-team/dontkillmy-app

Builds:
  - versionName: '1.8'
    versionCode: 20
    commit: 2e299ddb05d827ec43bbe688e24c94841022bea0
    subdir: app
    gradle:
      - yes

AutoUpdateMode: None
UpdateCheckMode: Tags
```
it fails with:
```
2020-07-27 12:24:27,879 DEBUG: buildserver > FAILURE: Build failed with an exception.
2020-07-27 12:24:27,880 DEBUG: buildserver >
2020-07-27 12:24:27,881 DEBUG: buildserver > * What went wrong:
2020-07-27 12:24:27,881 DEBUG: buildserver > Execution failed for task ':app:processReleaseResources'.
2020-07-27 12:24:27,882 DEBUG: buildserver > > A failure occurred while executing com.android.build.gradle.internal.tasks.Workers$ActionFacade
2020-07-27 12:24:27,883 DEBUG: buildserver >    > Android resource linking failed
2020-07-27 12:24:27,883 DEBUG: buildserver >      /home/vagrant/build/com.urbandroid.dontkillmyapp/app/src/main/res/layout/activity_rate.xml:19: AAPT: error: resource drawable/rate (aka com.urbandroid.dontkillmyapp:drawable/rate) not found.
2020-07-27 12:24:27,884 DEBUG: buildserver >
2020-07-27 12:24:27,885 DEBUG: buildserver >      /home/vagrant/build/com.urbandroid.dontkillmyapp/app/src/main/res/menu/menu_result.xml:9: AAPT: error: resource drawable/ic_help (aka com.urbandroid.dontkillmyapp:drawable/ic_help) not found.
```

--- a/README.md
+++ b/README.md
@@ -1,2 +1,23 @@
-# dontkillmy-app
-Tool for benchmarking nonstandard background processing limitations that are done by OEMs on different Android flavours.
+# DontKillMyApp
+
+Helps you set up your phone background tasks so that your apps can finally work for YOU even when not looking at the screen right now. See how is your phone doing and test different settings with DontKillMyApp benchmark.
+
+[<img src="https://fdroid.gitlab.io/artwork/badge/get-it-on.png"
+     alt="Get it on F-Droid"
+     height="80">](https://f-droid.org/packages/com.urbandroid.dontkillmyapp/)
+[<img src="https://play.google.com/intl/en_us/badges/images/generic/en-play-badge.png"
+     alt="Get it on Google Play"
+     height="80">](https://play.google.com/store/apps/details?id=com.urbandroid.dontkillmyapp)
+
+## Features:
+* DKMA benchmark: Measure how aggressively is your phone killing background apps
+* Guides: Get actionable steps to overcome most background process restrictions
+* Make a change:️ Help smartphones stay smart by sharing your benchmark report to dontkillmyapp.com
+
+DontKillMyApp is a tool for benchmarking nonstandard background processing limitations that are done by OEMs on different Android flavours. You can measure before setting up your phone, then go through the setup guides and benchmark again to see how much has your phone been slacking in the background.
+
+You can share your report through the app to the maintainers of the dontkillmyapp.com website who compile it and base the overall negative score on it.
+
+## How does the benchmark work?
+
+The app starts a foreground service with a wake lock and schedules repetitive task on the main thread, a custom thread executor and schedules regular alarms (AlarmManager.setExactAndAllowWhileIdle). Then it calculates executed vs. expected. That's it!

